docs/index.rst

```diff
@@ -567,13 +567,25 @@ Exotic Types
 (This script is complete, it should run "as is")
 
 
-StrictBool
-~~~~~~~~~~
+RelaxedBool and StrictBool
```

Comment:
Let's:
* change this to "RelaxedBool, StrictBool and bool" or just "Booleans"
* move the examples into a separate code example here
* add a warning explaining how bool behaviour changed in the new version.